pkr/driver/__init__.py

```diff
@@ -3,9 +3,22 @@
 
 """pkr drivers"""
 
-from stevedore.driver import DriverManager
+from importlib import import_module
+import pkgutil
+import os
 
+DRIVER_MAPPING = {
+    'compose': 'docker_compose',
+    'kubernetes': 'k8s',
+}
 
 def load_driver(driver_name):
     """Return the loaded driver"""
-    return DriverManager(namespace='drivers', name=driver_name).driver
+    if driver_name in DRIVER_MAPPING:
+        driver_name = DRIVER_MAPPING[driver_name]
+    module = import_module('pkr.driver.{}'.format(driver_name))
```

Comment:
Maybe a try / catch would be interesting here

Reply:
Also, you should consider using the optional package argument of this function.
Smth like : import_module(driver_name, 'pkr.driver')

Reply:
cf https://docs.python.org/3/library/importlib.html#importlib.import_module

Reply:
Fixed by calling pkr.driver.* on pkr.driver package